core/domain/topic_services_test.py

```diff
@@ -698,22 +698,6 @@ def test_manager_cannot_assign_roles(self):
         self.assertFalse(topic_services.check_can_edit_topic(
             self.user_b, topic_rights))
 
-    def test_reassigning_manager_role_to_same_user(self):
```

Comment:
I think this should be kept? At any rate, even if the behaviour is changed, tests should not just be deleted -- there needs to be coverage for the replacement functionality.

Reply:
Yeah, added the old check back, since it will no longer cause conflicts.